README.md

```diff
@@ -107,13 +107,15 @@ to see the usage of the `metadata add` command to upload metadata files.
 
 ## Environment Variables Reference
 
-| Name         | Description                                                                                                                                     |
-|--------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
-| `AT_TOKEN`   | AusTrakka auth token                                                                                                                            |
-| `AT_URI`     | URI for API endpoint                                                                                                                            |
-| `AT_ENV`     | Set to `dev` to log debugging                                                                                                                   |
-| `AT_LOG`     | Set to `file` to redirecting logging to a temp file                                                                                             |
-| `AT_CMD_SET` | Set to `austrakka-admin` to display admin commands (these will not actually run successfully unless you have an appropriate role on the server) |
+| Name             | Description                                                                                                                                     |
+|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| `AT_TOKEN`       | AusTrakka auth token                                                                                                                            |
+| `AT_URI`         | URI for API endpoint                                                                                                                            |
+| `AT_ENV`         | Set to `dev` to log debugging                                                                                                                   |
+| `AT_LOG`         | Set to `file` to redirecting logging to a temp file                                                                                             |
+| `AT_CMD_SET`     | Set to `austrakka-admin` to display admin commands (these will not actually run successfully unless you have an appropriate role on the server) |
+| `AT_VERIFY_CERT` | Verifies the cert used by the AusTrakka backend                                                                                                 |
```

Comment:
Why is AT_VERIFY_CERT a variable? When do we not want to verify cert?

Reply:
Development environments with self signed certs

Reply:
And why is HTTP2 experimental? Is it possible separate verify cert (which should be implemented with urgency) from http2 changes?

Reply:
We can release verify cert now and then just flip the http2 default later. It's experimental because I'm not 100% convinced of the performance improvements yet

Reply:
I though self-signed might be the issue. Is it simpler to separate cert verification on another clean branch? I see many changes here to drop call_api - for example. Which option will get the security fix out sooner with the least risks?

Reply:
> I though self-signed might be the issue. Is it simpler to separate cert verification on another clean branch? I see many changes here to drop call_api - for example. Which option will get the security fix out sooner with the least risks?

I'll make another branch with just cert verification